code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -513,7 +513,12 @@
 	caliber = "laser"
 	max_ammo = 20
 	multi_sprite_step = 5
+	w_class = WEIGHT_CLASS_NORMAL
 
+/obj/item/ammo_box/magazine/laser/ert //Used by red ERT. Keeps the size for them
+	name = "compact laser carbine projector magazine"
+	desc = "By use of bluespace technology, one can store many more lasers inside this magazine than otherwise would be reasonable."
```

Comment:
laser rounds instead of just "lasers"? shells? I dont know how its called for laser weaponary.

Reply:
Will look into it

Reply:
Done